src/authorizations/components/redesigned/EditTokenOverlay.tsx

```diff
@@ -115,59 +115,59 @@ const EditTokenOverlay: FC<Props> = props => {
                 />
               </Form.Element>
             </FlexBox>
-            <FlexBox.Child className="main-flexbox-child">
-              <FlexBox
-                margin={ComponentSize.Large}
-                justifyContent={JustifyContent.SpaceBetween}
-                direction={FlexDirection.Row}
-                stretchToFitWidth={true}
-                alignItems={AlignItems.Center}
-                className="flex-box-label"
-              >
-                <FlexBox.Child basis={40} grow={8}>
-                  <InputLabel size={ComponentSize.ExtraSmall}>
-                    Resources
-                  </InputLabel>
-                </FlexBox.Child>
-                <FlexBox.Child grow={1} className="flexbox-child-label-read">
-                  <InputLabel
-                    className="input-label-read"
-                    size={ComponentSize.ExtraSmall}
-                  >
-                    Read
-                  </InputLabel>
-                </FlexBox.Child>
-                <FlexBox.Child grow={1} className="flexbox-child-label-write">
-                  <InputLabel
-                    className="input-label-write"
-                    size={ComponentSize.ExtraSmall}
-                  >
-                    Write
-                  </InputLabel>
-                </FlexBox.Child>
-              </FlexBox>
-              <EditResourceAccordion
-                permissions={formatPermissionsObj(props.auth.permissions)}
-              />
-            </FlexBox.Child>
-            <Page.ControlBarCenter>
-              <FlexBox margin={ComponentSize.Medium}>
-                <Button
-                  color={ComponentColor.Tertiary}
-                  text="Cancel"
-                  onClick={handleDismiss}
-                  testID="token-cancel-btn"
-                />
-                <Button
-                  color={ComponentColor.Primary}
-                  text="Save"
-                  onClick={onSave}
-                  testID="token-save-btn"
-                  status={status}
-                />
-              </FlexBox>
-            </Page.ControlBarCenter>
           </Form>
```

Comment:
This change is just me moving the `</Form>` to line 118. It seems big but it's not really.